images/scripts/opensuse-micro.bootstrap

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
```

Comment:
Stupid question - what is micro and why does it need to have separate .fcc and .ign files and not use the same you used for tumbleweed?

Reply:
[microos](https://microos.opensuse.org) is the immutabile spin of opensuse. its is similar to fedora-coreos/rhel4edge and requires its separate fcc/ign file as without them the image does not boot, the common fcc/ign files have options that do not work with micro